packages/js/create-block-template/webpack.config.js.mustache

```diff
@@ -0,0 +1,94 @@
+const path = require( 'path' );
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+const WooCommerceDependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+
+const wcDepMap = {
+	'@woocommerce/blocks-checkout': [ 'wc', 'blocksCheckout' ],
+	'@woocommerce/blocks-registry': [ 'wc', 'wcBlocksRegistry' ],
+	'@woocommerce/settings': [ 'wc', 'wcSettings' ],
+};
+
+const wcHandleMap = {
+	'@woocommerce/blocks-registry': 'wc-blocks-registry',
+	'@woocommerce/blocks-checkout': 'wc-blocks-checkout',
+	'@woocommerce/settings': 'wc-settings',
+};
```

Comment:
`wcDepMap` can now be added to Woo's dependency extraction plugin directly

https://github.com/woocommerce/woocommerce/blob/918fa6408827a60075cb2d685952dd54b5a30fc5/packages/js/dependency-extraction-webpack-plugin/src/index.js#L23-L27

And same for `wcHandleMap` 

https://github.com/woocommerce/woocommerce/blob/918fa6408827a60075cb2d685952dd54b5a30fc5/packages/js/dependency-extraction-webpack-plugin/src/index.js#L44-L48